### PR TITLE
feat: Added Notifications APIs|| AuthenticatedUsers, ProjectMembers, Organi…

### PR DESCRIPTION
**Pull request by Durdush**
…zationMembers

### CODE REVIEW

File: src/main/java/com/crowdin/client/notifications/NotificationsApi.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public void sendNotificationToProjectMembers(Long projectId, NotificationRequestToProjectMember request) throws HttpException, HttpBadRequestException {
+        String builtUrl = String.format("%s/projects/%d/notify", this.url, projectId);
+        this.httpClient.post(builtUrl, request, new HttpRequestConfig(), Void.class);
+    }
+
+    /**
+     * @param request request object
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.notify.post" target="_blank"><b>API Documentation</b></a></li>
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/notifications/NotificationsApi.java:50]:
```suggestion
```

File: src/main/java/com/crowdin/client/notifications/NotificationsApi.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    public NotificationsApi(Credentials credentials, ClientConfig clientConfig) {
+        super(credentials, clientConfig);
+    }
+
+    /**
+     * @param request request object
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#tag/Notifications/api.projects.notification.post" target="_blank"><b>API Documentation</b></a></li>
+     * <li><a href="https://developer.crowdin.com/enterprise/api/v2/#tag/Notifications/api.projects.notification.post" target="_blank"><b>Enterprise API Documentation</b></a></li>
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/notifications/NotificationsApi.java:26]:
```suggestion
```

File: src/main/java/com/crowdin/client/notifications/NotificationsApi.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+     * </ul>
+     */
+    public void sendNotificationToAuthenticatedUser(NotificationRequestToAuthenticatedUsers request) throws HttpException, HttpBadRequestException {
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/notifications/NotificationsApi.java:29]:
```suggestion
    public void sendNotificationToAuthenticatedUser(SendNotificationToAuthenticatedUserRequest request) throws HttpException, HttpBadRequestException {
```

File: src/main/java/com/crowdin/client/notifications/NotificationsApi.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+    /**
+     * @param projectId Project Identifier
+     * @param request request object
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.projects.notify.post" target="_blank"><b>API Documentation</b></a></li>
+     * <li><a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.notify.post" target="_blank"><b>Enterprise API Documentation</b></a></li>
+     * </ul>
+     */
+    public void sendNotificationToProjectMembers(Long projectId, NotificationRequestToProjectMember request) throws HttpException, HttpBadRequestException {
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/notifications/NotificationsApi.java:42]:
```suggestion
    public void sendNotificationToProjectMembers(Long projectId, SendNotificationToProjectMembersRequest request) throws HttpException, HttpBadRequestException {
```

File: src/main/java/com/crowdin/client/notifications/NotificationsApi.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    /**
+     * @param request request object
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.notify.post" target="_blank"><b>API Documentation</b></a></li>
+     * <li><a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.notify.post" target="_blank"><b>Enterprise API Documentation</b></a></li>
+     * </ul>
+     */
+    public void sendNotificationToOrganizationMembers(NotificationRequestToOrganizationMembers request) throws HttpException, HttpBadRequestException {
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/notifications/NotificationsApi.java:54]:
```suggestion
    public void sendNotificationToOrganizationMembers(SendNotificationToOrganizationMembersRequest request) throws HttpException, HttpBadRequestException {
```

File: src/test/resources/api/notifications/notificationRequestToOrganizationMembersByRole.json [hunk 6]
```diff
@@ -0,0 +1,4 @@
+{
+  "role": "Admin",
```
Comment (andrii-bodnar) [inline src/test/resources/api/notifications/notificationRequestToOrganizationMembersByRole.json:2]:
```suggestion
  "role": "owner",
```

File: src/test/resources/api/notifications/notificationRequestToProjectMembersByRole.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
+  "role": "Admin",
```
Comment (andrii-bodnar) [inline src/test/resources/api/notifications/notificationRequestToProjectMembersByRole.json:2]:
```suggestion
  "role": "owner",
```

File: src/test/resources/api/notifications/notificationRequestToOrganizationMembersByUserIds.json
Comment (andrii-bodnar) [inline src/test/resources/api/notifications/notificationRequestToOrganizationMembersByUserIds.json:1]: Please also rename the test files according to the names of the request classes

Comment (Durdush) [reply src/test/resources/api/notifications/notificationRequestToOrganizationMembersByUserIds.json:1]: Hi @andrii-bodnar, modified and committed code as per review. Please review it now.

**Review by andrii-bodnar — APPROVED** [review]
@Durdush looks good to me, thank you!

**Pull request merged**